### DISCUSSION TITLE
Unable to log in to Microsoft account

### DISCUSSION THREAD

**Discussion by cooptest** [microsoft/semantic-kernel#943] [Q&A]
I built it on a public network server, dotnet and yarn run successfully, but the front end cannot log in, and the click does not respond
<img width="889" alt="1" src="https://github.com/microsoft/semantic-kernel/assets/133195857/f6f1bc85-caab-4d07-8157-b846d43ba639">

My redirection ip in Azure is public IP: https://x.x.x.x:3000

<img width="644" alt="2" src="https://github.com/microsoft/semantic-kernel/assets/133195857/48e89ba5-826f-408f-bf1d-31260d0ba97a">

Azure does not support https://x.x.x.x:3000

I set it to http://localhost:3000/https://localhost:3000/https:x.x.x.x:3000 and I can’t log in

**Comment by glahaye** [comment]
@teresaqhoang Do you know anything about this?

**Comment by teresaqhoang** [comment]
Hi @cooptest, can you add http://localhost:3000 without the s as a redirect URL and see what the behavior is? The WebApp currently doesn't support https.

**Comment by cooptest** [comment]
@teresaqhoang http://localhost:3000 can run successfully

But I want to deploy the test on the Internet, does it not support deployment on the Internet now?

Even if I deploy it on the intranet, it must be accessed with http://localhost:3000, not with the intranet ip.

**Comment by teresaqhoang** [comment]
Hi @cooptest, sorry for the late response, we don't currently support deployment on the Internet but it's something our team is actively working on! Will defer to @glahaye to provide updates.

**Comment by teresaqhoang** [comment]
Closing as dupe: https://github.com/microsoft/semantic-kernel/issues/944
